values-tls.yaml

```diff
@@ -0,0 +1,172 @@
+# Default values for infinispan-helm-charts.
```

Comment:
Instead of providing a full values file that needs to be updated whenever new features are added or versions change, we could use a tool like [yq](https://github.com/mikefarah/yq) or Kustomize to simply patch `values.yaml` with the changes required for the test.

Reply:
I added the yq patch, but I think it would be great to provide the final tls_values.yaml result, people may don't want to deal with yq. wdyt?

Reply:
well I mean a ready to use example is better than a patch

Reply:
nice, get ride of yq then